sentry_sdk/integrations/aiohttp.py

```diff
@@ -113,34 +112,31 @@ async def sentry_app_handle(self, request, *args, **kwargs):
                     scope.add_event_processor(_make_request_processor(weak_request))
 
                     headers = dict(request.headers)
-                    transaction = continue_trace(
-                        headers,
-                        op=OP.HTTP_SERVER,
-                        # If this transaction name makes it to the UI, AIOHTTP's
-                        # URL resolver did not find a route or died trying.
-                        name="generic AIOHTTP request",
-                        source=TRANSACTION_SOURCE_ROUTE,
-                        origin=AioHttpIntegration.origin,
-                    )
-                    with sentry_sdk.start_transaction(
-                        transaction,
-                        custom_sampling_context={"aiohttp_request": request},
-                    ):
-                        try:
-                            response = await old_handle(self, request)
-                        except HTTPException as e:
-                            transaction.set_http_status(e.status_code)
-                            raise
-                        except (asyncio.CancelledError, ConnectionResetError):
-                            transaction.set_status(SPANSTATUS.CANCELLED)
-                            raise
-                        except Exception:
-                            # This will probably map to a 500 but seems like we
-                            # have no way to tell. Do not set span status.
-                            reraise(*_capture_exception())
-
-                        transaction.set_http_status(response.status)
-                        return response
+                    with sentry_sdk.continue_trace(headers):
+                        with sentry_sdk.start_transaction(
```

Comment:
I wonder whether it might make sense to introduce a new convenience function that would continue the trace and start a span in one call, so that these two `with` context managers could be combined into a single `with continue_trace_start_span` (perhaps with better naming) statement.